icubam/predicu/data.py

```diff
@@ -106,16 +106,19 @@ def load_bedcounts(
 
 
 def load_icubam(cached_data=None, api_key=None, icubam_host=None, clean=True):
-  if api_key is None or icubam_host is None:
-    raise RuntimeError("Provide API key and host to download ICUBAM data")
+  if cached_data is not None and 'raw_icubam' in cached_data:
```

Comment:
What happens here if there is no cached data for the API endpoint?

Reply:
this `cached_data` dict is just a trick not to load the data from the API (`all_bedcounts` request) and use the data in the `cached_data` dict instead. see the call to `load_bedcounts` in the handler: i've added a comment about that there.